packages/compiler/test/bad-styles/unclosed-style.ts

```diff
@@ -0,0 +1,15 @@
+import { test } from 'uvu';
+import * as assert from 'uvu/assert';
+import { parse } from '@astrojs/compiler';
+
+test('can compile unfinished style', async () => {
+  let error = 0;
+  try {
+    await parse(`<style>`);
+  } catch (e) {
+    error = 1;
+  }
+  assert.equal(error, 0, 'Expected to compile with unfinished style.');
```

Comment:
wouldn't it make sense to also assert on the output if this is expected to not error?